README.md

```diff
@@ -1 +1,89 @@
+| ovirt_engine_setup_db_port            | 5432                  | Engine database port. |
+| ovirt_engine_setup_db_name            | engine                | Engine database name. |
+| ovirt_engine_setup_db_user            | engine                | Engine database user. |
+| ovirt_engine_setup_db_password        | UNDEF                 | Engine database password. |
+| ovirt_engine_setup_dwh_db_host        | localhost             | IP address or host name of a PostgreSQL server for DWH database. By default the DWH database will be configured on the same host as Engine. |
+| ovirt_engine_setup_dwh_db_port        | 5432                  | DWH database port. |
+| ovirt_engine_setup_dwh_db_name        | ovirt_engine_history  | DWH database name. |
+| ovirt_engine_setup_dwh_db_user        | ovirt_engine_history  | DWH database user. |
+| ovirt_engine_setup_dwh_db_password    | UNDEF                 | DWH database password. |
+| ovirt_engine_setup_provider_ovn_username | admin@internal     | Username for OVN. |
+| ovirt_engine_setup_provider_ovn_password | UNDEF              | Password for OVN. |
+| ovirt_engine_setup_organization       | UNDEF                 | Organization name for certificate. |
+
+* ISO domain related options:
+
+| Name                              | Default value         |  Description                               |
+|-----------------------------------|-----------------------|--------------------------------------------|
+| ovirt_engine_setup_configure_iso_domain | False                 | If `True`, ISO storage domain will be configured on Engine host.   |
+| ovirt_engine_setup_iso_domain_path      | /var/lib/exports/iso  | Path to a directory on Engine host to store data for ISO storage domain. |
+| ovirt_engine_setup_iso_domain_name      | IDO_DOMAIN            | Name of ISO domain.                         |
+| ovirt_engine_setup_iso_domain_acl       | 0.0.0.0/0.0.0.0(rw)   | ACL permissions for ISO domain mount point. |
+
+* Configure firewall
+
+| Name                             | Default value  |  Description |
+|----------------------------------|----------------|---------------------------------------------------------------------------------------------------------------------|
+|  ovirt_engine_setup_firewall_manager   | firewalld      | Specify the type of firewall manager to configure on Engine host, following values are availableL: `firewalld`,`iptables` or empty value to skip firewall configuration. |
+
+* Update
+
+| Name                  | Default value         |  Description                                              |
+|-----------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_update_packages   | False        | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sence if Engine is already installed. |
```

Comment:
As mentioned before I'd change default to true, as the most common use cases to execute this role are to perform new installation (updating setup packages should not break anything) or upgrade (updating setup packages is a must for upgrade)

Reply:
Makes thanks, done.